vocabularyEntryDetails.md

```diff
@@ -0,0 +1,165 @@
+# Vocabulary Entries Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+All the endpoints here described are restricted to authenticated users.
+
+## Main Endpoint
+**/api/integration/vocabularyEntryDetails**   
+
+Not allowed. Only subset of vocabulary entries can be retrieved using specific filters see below
+
+## Single Authority
+**/api/integration/vocabularyEntryDetails/<vocabulary-name><:entry-id>**
+
+Provide detailed information about a specific vocabulary entry. The JSON response document is as follow
+```json
+{
+      "id" : "srsc:SCB110",
+      "value" : "Religion/Theology",
```

Comment:
In this endpoint `value` seems to be the "display value".  But in `/vocabularies/[:name]/entries`, the `value` is something like 'Research Subject Categories::SOCIAL SCIENCES::Social sciences::Social work::Youth research".   I think we should be making these two endpoints more similar (to avoid confusion).  So, this example should use `value` in a similar way, and `display` should be added to this example.